src/PKSim.UI/Views/Applications/ApplicationParametersView.cs

```diff
@@ -49,10 +46,10 @@ public ApplicationParametersView(IImageListRetriever imageListRetriever)
          mainView.ShowColumnHeaders = false;
 
          gridViewParameters.OptionsView.ShowGroupPanel = false;
+         gridViewParameters.MultiSelect = false;
```

Comment:
This was enough to make it work. Using Multiselect true created some issue with background color. Good enough for now